packages/shared/src/graphql/sources.ts

```diff
@@ -142,3 +142,18 @@ export const SIMILAR_SOURCES_QUERY = gql`
     }
   }
 `;
+
+export enum PublicSquadRequestStatus {
+  Pending = 'pending',
```

Comment:
Didn't we type these somewhere else? maybe not, I remember seeing something like this, but not sure so just checking

Reply:
I'm not sure anymore. I think we only did it on the API side :)